app/views/projects/show.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="container">
+  <div class=" col-md-12 col-lg-12 ">
+    <h2 class="text-primary"><strong>Basic information</strong> </h2>
+    <table class="table table-bordered text-size">
+      <thead>
```

Comment:
The structure below not making sense to wrap it in `thead` pls change it to <tr>